### ISSUE TITLE
Typescript config available?

### ISSUE THREAD

**Issue by qasamppa** [playwright-community/eslint-plugin-playwright#176]
Could you write good instructions for Typescript Playwright users? There are couple extra steps to get it working with TS, I think.

**Comment by mskelton** [comment]
This plugin doesn't require any specific setup for TypeScript. Could you share more details about what isn't working?

**Comment by ychaudhari** [comment]
I get a  `error  Parsing error: Unexpected token :` on following line in the `sample-test.test.ts` file:

```typescript
const expected: Array<string> = [
  'some-value 1',
  'some-value 2',
];
```

**Comment by mskelton** [comment]
More than likely you need haven't configured typescript-eslint.

https://typescript-eslint.io/

This is not an issue with the Playwright plugin, just that you haven't configured for TypeScript.
